js/typed.js

```diff
@@ -0,0 +1,438 @@
+// The MIT License (MIT)
```

Comment:
I wonder about adding a whole library for a small function, but let's see how it looks.